### PR TITLE
docker-entrypoint-initdb.d init scripts

### PR DESCRIPTION
**Pull request by fboucquez**
Fixes https://github.com/nemtech/symbol-bootstrap/issues/54

### CODE REVIEW

File: config/docker/mongo/mongoDbPrepare.js [hunk 1]
```diff
@@ -1,80 +1,90 @@
+print(`DB Prepare`);
```
Comment (Wayonb) [inline config/docker/mongo/mongoDbPrepare.js:1]: maybe I missed it, where do you specify the database to create the collections in?
There is an env MONGO_INITDB_DATABASE that should be set.

**Review by Wayonb — CHANGES_REQUESTED** [review]

File: config/docker/mongo/mongoDbPrepare.js (hunk 1 above)
Comment (fboucquez) [reply config/docker/mongo/mongoDbPrepare.js:1]: You are right, I've just found out that the server inits the database itself, that's why it was working when I pushed the branch. Why is compose creating the tables if the server can do it? We would avoid maintaining the schemas on bootstrap.

Comment (fboucquez) [reply config/docker/mongo/mongoDbPrepare.js:1]: it would also boot faster...

Comment (Wayonb) [reply config/docker/mongo/mongoDbPrepare.js:1]: @Jaguar0625 @gimer Thoughts?

**Comment by fboucquez** [comment]
Hi @Wayonb 

For now can use the copied db mongo script until we have server images with the scripts built-in. 

I've created the issue:

https://github.com/nemtech/symbol-bootstrap/issues/58

This issue depends on the server docker image to include the scripts.

**Review by Wayonb — APPROVED** [review]

**Pull request merged**